### DISCUSSION TITLE
`Checkbox` styles are not applied when it is inside `CheckboxGroup`

### DISCUSSION THREAD

**Discussion by I-am-tk** [mantinedev/mantine#1173] [Q&A] [answered]
I styled the `Checkbox` component using the `styles` property in `MantineProvider`, but those styles are not applied to the `Checkbox` component when it is placed inside the `Checkbox` group component.

Take a look at the code https://codesandbox.io/s/epic-franklin-uirbug?file=/src/App.tsx. Is there a way to do this?

**Comment by rtivital** [comment, marked as answer]
Seems like a bug, I'll fix it in one of the next releases

**Comment by lyndseysc** [comment]
did this fix get released? 
experiencing the same problem with the `size` and `width` Checkbox props within a Checkbox group not being applied. Version is`"@mantine/core": "^5.10.3". Workaround is just sizing within the Checkbox group.

**Comment by korayaggul** [comment]
hello @rtivital the problem still continues 

https://github.com/user-attachments/assets/a1c45a51-6154-4710-a525-00ab47297535
